packages/tooltip-react/src/Tooltip.tsx

```diff
@@ -45,17 +46,22 @@ export const Tooltip = ({ content, initialPlacement = "top", className }: Toolti
     const { getReferenceProps, getFloatingProps } = useInteractions([
         useClick(context),
         useDismiss(context, { referencePress: false }),
-        useHover(context),
-        useFocus(context),
+        useFocus(context, { enabled: open }), // Lukk når fokus flyttes vekk
```

Comment:
> Lukk når fokus flyttes vekk

Det skjer ikke for meg i Safari 16.1 på https://jokul.fremtind.no/preview/fix/tooltip/komponenter/tooltip, hvis det er meningen.

![](https://user-images.githubusercontent.com/1223410/205009817-5c0b9e25-a043-49b7-81ff-3b717c6478ba.gif)

Reply:
Huh, works on my machine! Skal sjekke om det har sneket seg inn noe rart 😊 Er meningen at det skal oppføre seg som det står..

Reply:
@wkillerud kan du teste igjen nå? Har ikke gjort noe mer enn å merge inn main, men det ser ut til å funke i preview 🤷‍♂️

Reply:
Ser det samme som deg @piofinn, funker nå 🤷